README.md

```diff
@@ -148,53 +148,56 @@ import HyperwalletUISDK
 ### Lists the user's transfer methods
 The user can deactivate and add a new transfer method.
 ```swift
-let listTransferMethodController = HyperwalletUI.shared.listTransferMethodController()
```

Comment:
Add the documentation on how to add only specific module for cocoapod and carthage in the cocoapod and carthage section. Add some details about how it will work. For cocoapod, If user wants all the modules then what they need to do and if they want to use only specific module then what they need to do. Same goes for carthage.